build/wix/subdirs.txt

```diff
@@ -0,0 +1,7 @@
+controllers
```

Comment:
I did see that in one of the files, yes. So maybe this file isn't needed at all now?